test/MutationTestInfrastructure/DynamoViewModelMutation.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Dynamo.Models;
+using Dynamo.ViewModels;
+using System;
+
+namespace MutationTestInfrastructure
```

Comment:
This doesn't really below in test to be honest. It's a utility class more than anything else. Could we move it to tools?